cluster/ephemeral-provider-common.sh

```diff
@@ -29,11 +29,11 @@ function _registry_volume() {
 }
 
 function _add_common_params() {
-    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory 4096M --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
+    local params="--nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE}M --cpu 4 --random-ports --background --prefix $provider_prefix --registry-volume $(_registry_volume) kubevirtci/${image} ${KUBEVIRT_PROVIDER_EXTRA_ARGS}"
     if [[ $TARGET =~ windows.* ]]; then
-        params="--memory 8192M --nfs-data $WINDOWS_NFS_DIR $params"
```

Comment:
With the new location only stdci would set the correct memory size. The value needs to be either part of `automation/test.sh` or part of this file.

Reply:
+1 on test.sh, line 55 looks like a best place

Reply:
Overall, I think that just fixing the if statement to not include `--memory` two times here would have been the best thing to do. The build environment is not a general-purpose tool and the scripts *support* the `TARGET` env vars explicitly. I don't see much of a reason to play with the memory values, since CI should test over- and over that the memory size is sufficient. Thoughts?

Reply:
but then all tests which runs this script will have 8GB of memory/node. I think line 100 will be better.

Reply:
If 8GB are not needed,we can reduce the value. The only thing we want to achieve is this: "Every node of the test environment should be capable of running a windows machine".

Reply:
@cynepco3hahue I remember we had to choose 8gb to make some tests pass, but it looks like we by accident always used 4gb and the tests work fine. Can that be?

Reply:
I wonder if some of our flaky tests in openshift will go away once we fix the `if` statement.